lib/node/Creator.js

```diff
@@ -14,22 +14,20 @@ const opFns = {
   '^': 'pow',
 };
 
-const operatorNode = (op, fn, args) => {
+const operatorNode = (op, fn, args, implicit) => {
   return {
     type: 'OperatorNode',
     op: op,
     fn: fn,
     args: args,
+    implicit: implicit,
   };
 };
 
 const NodeCreator = {
   operator (op, args, implicit=false) {
     if (op in opFns) {
-      if (op === '-' && args.length === 1) {
-        debugger;
```

Comment:
I have no idea why we chose to put `debugger` here, but it's true that if it's actually a unary minus, we shouldn't be using this constructor and should use the `unaryMinus` constructor instead. It'd be worth throwing an error here